x-pack/plugins/actions/server/lib/action_executor.ts

```diff
@@ -120,6 +120,8 @@ export class ActionExecutor {
     }
 
     const actionLabel = `${actionTypeId}:${actionId}: ${name}`;
+    logger.debug(`executing action ${actionLabel} at ${new Date().toISOString()}`);
```

Comment:
With the logger prefixing all the logs with a timestamp, is there a reason for the extra `at ${new Date().toISOString()}` here?